docs/installation/presto-server-installation.rst

```diff
@@ -21,8 +21,12 @@ Presto! Presto is now installed on the coordinator and workers specified in your
 The default port for Presto is 8080.  If that port is already in use on your cluster, you will not be able to start Presto.
 In order to change the port that Presto uses, proceed to :ref:`presto-port-configuration-label`.
 
+There are additional configuration properties described at :ref:`presto-configuration-label` that
+must be changed for optimal performance. These configuration changes can be done either
+before or after starting the Presto server and running queries for the first time.
```

Comment:
I suppose. That page that's linked to very clearly mentions the need to restart the server, but doesn't hurt to add it. Most of this doc work will be thrown away once Brenda finishes, so I do'nt want to spend too much time tweaking it.